src/interface_r/R/auto_generated_wrappers.R

```diff
@@ -446,57 +361,42 @@ h2o4gpu.kmeans <- function(
     algorithm = algorithm,
     gpu_id = as.integer(gpu_id),
     n_gpus = as.integer(n_gpus),
-    do_checks = as.integer(do_checks),
-    backend = backend
+    do_checks = as.integer(do_checks)
   )
   h2o4gpu_model(model, NULL)
 }
 
 #' @export
 h2o4gpu.pca <- function(
 	n_components = 2L,
-	copy = TRUE,
-	whiten = FALSE,
-	svd_solver = "arpack",
-	tol = 0.0,
-	iterated_power = "auto",
-	random_state = NULL,
-	verbose = FALSE,
-	backend = "auto") {
+	whiten = FALSE) {
 
-  model <- h2o4gpu$PCA(
+  model <- h2o4gpu$PCAH2O(
     n_components = as.integer(n_components),
-    copy = copy,
-    whiten = whiten,
-    svd_solver = svd_solver,
-    tol = tol,
-    iterated_power = iterated_power,
-    random_state = as_nullable_integer(random_state),
-    verbose = verbose,
-    backend = backend
+    whiten = whiten
   )
   h2o4gpu_model(model, NULL)
 }
 
 #' @export
 h2o4gpu.truncated_svd <- function(
 	n_components = 2L,
-	algorithm = "cusolver",
-	n_iter = 5L,
+	algorithm = "power",
+	n_iter = 100L,
```

Comment:
Why does "TruncatedSVDH2O" have different default vaules for `algorithm` and `n_iter` as compared to the "TruncatedSVD" class?  This is not good because if you use this methods in R and Python, they will produce different results by default.

Reply:
I have updated this